packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -45,32 +45,32 @@ export class TopNavigation {
   @Element() el: HTMLIcTopNavigationElement;
 
   /**
-   *  Title to display
+   * The app title to be displayed.
    */
   @Prop() appTitle!: string;
 
   /**
-   *  The url to navigate to when title is clicked
+   *  The url to navigate to when the app title is clicked.
    */
   @Prop() href: string = "/";
 
   /**
-   *  Version info to display
+   * The version info to be displayed.
    */
   @Prop() version: string = "";
 
   /**
-   *  Status info to display
+   * The status info to be displayed.
    */
   @Prop() status: string = "";
 
   /**
-   * The alignment of the top nav content
+   * The alignment of the top nav content.
```

Comment:
Might be good to refer to it as its full name
```suggestion
   * The alignment of the top navigation content.
```